Sources/MapboxNavigation/CarPlayManager.swift

```diff
@@ -735,15 +782,3 @@ internal class MapTemplateProvider: NSObject {
         return CPMapTemplate()
     }
 }
-
-#else
-/**
- CarPlay support requires iOS 12.0 or above and the CarPlay framework.
- */
-public class CarPlayManager: NSObject {
```

Comment:
Finally, jazzy will stop warning about missing USRs for these conditionally compiled symbols. 😌

/ref #2078